WooCommerce/WooCommerceTests/ViewRelated/Blaze/BlazeEditAdViewModelTests.swift

```diff
@@ -0,0 +1,187 @@
+        // When
+        sut.tagline = sampleString(length: 1)
+
+        // Then
+        let expectedString = String(format: BlazeEditAdViewModel.Localization.LengthLimit.plural, 31)
+        XCTAssertEqual(sut.taglineFooterText, expectedString)
+    }
+
+    func test_tagline_footer_text_is_singular_when_one_character_remaining() {
+        // Given
+        let sut = BlazeEditAdViewModel(siteID: 123,
+                                       adData: sampleAdData,
+                                       onSave: { _ in })
+        // When
+        sut.tagline = sampleString(length: 31)
+
+        // Then
+        let expectedString = String(format: BlazeEditAdViewModel.Localization.LengthLimit.singular, 1)
+        XCTAssertEqual(sut.taglineFooterText, expectedString)
+    }
+
+    func test_tagline_footer_text_is_singular_when_zero_character_remaining() {
+        // Given
+        let sut = BlazeEditAdViewModel(siteID: 123,
+                                       adData: sampleAdData,
+                                       onSave: { _ in })
+        // When
+        sut.tagline = sampleString(length: 32)
+
+        // Then
+        let expectedString = String(format: BlazeEditAdViewModel.Localization.LengthLimit.plural, 0)
+        XCTAssertEqual(sut.taglineFooterText, expectedString)
+    }
+
+    func test_tagline_footer_text_shows_error_when_tagline_is_empty() {
+        // Given
+        let sut = BlazeEditAdViewModel(siteID: 123,
+                                       adData: sampleAdData,
+                                       onSave: { _ in })
+
+        // When
+        sut.tagline = ""
+
+        // Then
+        XCTAssertEqual(sut.taglineFooterText, BlazeEditAdViewModel.Localization.taglineEmpty)
+    }
+
+    // MARK: Description
+
+    func test_description_footer_text_is_plural_when_multiple_characters_remaining() {
+        // Given
+        let sut = BlazeEditAdViewModel(siteID: 123,
+                                       adData: sampleAdData,
+                                       onSave: { _ in })
+        // When
+        sut.description = sampleString(length: 1)
+
+        // Then
+        let expectedString = String(format: BlazeEditAdViewModel.Localization.LengthLimit.plural, 139)
+        XCTAssertEqual(sut.descriptionFooterText, expectedString)
+    }
+
+    func test_description_footer_text_is_singular_when_one_character_remaining() {
+        // Given
+        let sut = BlazeEditAdViewModel(siteID: 123,
+                                       adData: sampleAdData,
+                                       onSave: { _ in })
+        // When
+        sut.description = sampleString(length: 139)
+
+        // Then
+        let expectedString = String(format: BlazeEditAdViewModel.Localization.LengthLimit.singular, 1)
+        XCTAssertEqual(sut.descriptionFooterText, expectedString)
+    }
+
+    func test_description_footer_text_is_singular_when_zero_character_remaining() {
```

Comment:
Same case here, I think it should be _plural_

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/11612/commits/b4bd01acad511c35068f8fb73c926329ec43a623